src/cosmosdb-continuous-backup/azext_cosmosdb_pitr/azext_metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+    "azext.isPreview": true,
+    "azext.minCliCoreVersion": "2.0.67",
+    "azext.maxCliCoreVersion": "2.16.1"
```

Comment:
may I know why you limit max version here?

Reply:
Good point. I dont need to limit the max version. I thought it is mandatory field. 
I removed it now